packages/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/hardened.yaml

```diff
@@ -1,6 +1,6 @@
 {{- $namespaces := dict "_0" .Release.Namespace -}}
-{{- if and .Values.grafana.enabled (or .Values.grafana.sidecar.dashboards.enabled .Values.grafana.defaultDashboardsEnabled) -}}
-{{- $_ := set $namespaces "_1" .Values.grafana.sidecar.dashboards.searchNamespace -}}
+{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboards.enabled (not .Values.grafana.defaultDashboards.useExistingNamespace) -}}
```

Comment:
Dropping `.Values.grafana.sidecar.dashboards.enabled` was intentional since the Job that is deployed to patch the default service account in the namespace is tied to whether or not we introduce it.

Since we modified the logic for deciding whether or not to deploy the namespace, this one is modified in conjunction with that.